packages/pytest-simcore/src/pytest_simcore/docker_swarm.py

```diff
@@ -33,11 +33,12 @@ def keep_docker_up(request) -> bool:
 @pytest.fixture(scope="module")
 def docker_swarm(
     docker_client: docker.client.DockerClient, keep_docker_up: Iterator[bool]
-) -> None:
+) -> Iterator[None]:

```

Comment:
it might be me... but it returns nothing... why the iterator?

Reply:
because it ``yields``